src/QMCDrivers/QMCCostFunctionBase.cpp

```diff
@@ -562,7 +562,7 @@ void QMCCostFunctionBase::updateXmlNodes()
   std::map<std::string, xmlNodePtr>::iterator pit(paramNodes.begin()), pit_end(paramNodes.end());
   while (pit != pit_end)
   {
-    Return_t v = OptVariablesForPsi[(*pit).first];
+    Return_t v = std::real( OptVariablesForPsi[(*pit).first] );
```

Comment:
Please leave a FIXME note indicating that real value is forced here.

Reply:
FIXME note added.